framework/python/src/common/testreport.py

```diff
@@ -134,6 +136,12 @@ def to_json(self):
 
   def from_json(self, json_file):
 
+    # Version added in v1.3-alpha
+    if 'testrun' in json_file and 'version' in json_file['testrun']:
+      self._version = json_file['testrun']['version']
+    else:
+      self._version = '1.3-alpha'
+

```

Comment:
Could we pass the session or version into the constructor for the TestReport object to set a default?

Reply:
The version here is the version which was used when the report was executed, not the current version of Testrun. I have replaced the fallback version with 'Unknown'